sql/wsrep_high_priority_service.cc

```diff
@@ -483,6 +484,7 @@ int Wsrep_high_priority_service::apply_toi(const wsrep::ws_meta &ws_meta,
            (long long)wsrep_thd_trx_seqno(m_thd));
   WSREP_DEBUG("%s", m_thd->wsrep_info);
   thd_proc_info(thd, m_thd->wsrep_info);
+  thd->set_command(COM_SLEEP);
```

Comment:
Can you please check if the same fix is required for NBO as well?

Reply:
NBO was not affected in the way reported in the PXC-4225, but I improved thread state reporting for it.